packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTableSet.tsx

```diff
@@ -193,14 +193,14 @@ export class ApiReferencesTableSet extends React.Component<IApiReferencesTableSe
     const enumMembers: IApiEnumProperty[] = [];
 
     const members: IEnumTableRowJson[] = table.members as IEnumTableRowJson[];
-    for (let k = 0; k < members.length; k++) {
+    members.map((member: IEnumTableRowJson) => {
       // each member within the enum
       enumMembers.push({
-        description: members[k].description,
-        name: members[k].name,
-        value: members[k].value
+        description: member.description,
+        name: member.name,
+        value: member.value
```

Comment:
nit, but you might consider destructuring in these blocks and using default values as in the case below with empty string. Purely a style comment tho.